src/sentry/api/bases/organization_integrations.py

```diff
@@ -20,6 +24,9 @@ class OrganizationIntegrationBaseEndpoint(IntegrationEndpoint):
 
     permission_classes = (OrganizationIntegrationsPermission,)
 
+    def fetch_org(self, slug: str) -> Union[Organization, RpcOrganizationSummary]:
```

Comment:
```suggestion
    def fetch_org(self, slug: str) -> Organization:
```

🤞 are we able to narrow the union here?

Reply:
I struggled with this question. It's an inherited method that actually swaps return values repeatedly in the inheritance tree, so while any particular implementation can narrow the return type I wasn't sure if mypy would be happy about that.